XTS/raw-xts-api-demo/pom.xml

```diff
@@ -80,4 +64,21 @@
         <module>ear</module>
         <module>test</module>
     </modules>
+
+    <profiles>
+<!--     jdk17 profile is defined because org.jboss.arquillian.graphene.shaded.net.sf.cglib.core.ReflectUtils needs -->
+<!--     java.lang add-opens in order to work with JDK17  -->
+<!--     see https://docs.oracle.com/en/java/javase/16/migrate/migrating-jdk-8-later-jdk-releases.html#GUID-7BB28E4D-99B3-4078-BDC4-FC24180CE82B -->
+     <profile>
+          <id>jdk17</id>
+          <activation>
+            <jdk>[17,)</jdk>
+          </activation>
+          <properties>
+             <modular.jdk.args>
+              --add-opens=java.base/java.lang=ALL-UNNAMED
+            </modular.jdk.args>
+          </properties>
+      </profile>
+  </profiles>
```

Comment:
Have you tried to update `org.jboss.arquillian.graphene`? `2.0.3.Final` was released in Jun 2014. I would try graphene version `2.5.4` and possibly `3.0.0-alpha.3`

Reply:
Thanks @jmfinelli for your suggestion! I managed to update the quickstart as much as I could, but the dependency need to be a fixed version to work together. Nevertheless I managed to make it run without the need to have the modular jvm arguments for jdk 17.